README.md

```diff
@@ -25,132 +21,112 @@ Package.onUse(function (api) {
-<table>
-<thead>
-<tr><th>Meteor Version</th><th>Recommended fourseven:scss version</th></tr>
-</thead>
-<tbody>
-<tr><td>1.0 - 1.1</td><td>3.2.0</td></tr>
-<tr><td>1.2 - 1.3.1</td><td>3.4.2</td></tr>
-<tr><td>1.3.2+</td><td>3.8.0_1</td></tr>
-<tr><td>1.4.0</td><td>3.8.1</td></tr>
-<tr><td>1.4.1+</td><td>4.5.4</td></tr>
-<tr><td>1.6+</td><td>4.12.0</td></tr>
-</tbody>
-</table>
-
-Since `meteor 1.4.1+` (`fourseven:scss 3.9.0+`), we do not have prebuild binaries anymore. You are required to set up the [required toolchain](https://github.com/nodejs/node-gyp) yourselves.
+
+- This plugin has been verified to work with Meteor 2.10 and later.
+- It offers full compatibility with Meteor 3.
+- It uses the [Dart Sass](https://www.npmjs.com/package/sass) npm package, meaning it only works on systems supported by Dart Sass: Windows, Mac OS, and Linux.
 
 ## Usage
-Without any additional configuration after installation, this package automatically finds all `.scss` and `.sass` files in your project, compiles them with [node-sass](https://github.com/sass/node-sass), and includes the resulting CSS in the application bundle that Meteor sends to the client. The files can be anywhere in your project.
 
-### File types
+After installation, this package automatically finds all `.scss` and `.sass` files in your project, compiles them with [Dart Sass](https://www.npmjs.com/package/sass), and includes the resulting CSS in the application's client bundle. These files can be located anywhere in your project.
 
-There are two different types of files recognized by this package:
+### File Types
 
-- Sass sources (all `*.scss` and `*.sass` files that are not imports)
-- Sass imports/partials, which are:
-  * files that are prefixed with an underscore `_`
-  * marked as `isImport: true` in the package's `package.js` file:
-    `api.addFiles('x.scss', 'client', {isImport: true})`
-  * Starting from Meteor 1.3, all files in a directory named `imports/`
+There are two main types of Sass files handled by this package:
 
-The source files are compiled automatically (eagerly loaded). The imports are not loaded by
-themselves; you need to import them from one of the source files to use them.
+- **Sass source files**: These are the `*.scss` and `*.sass` files that are not imports.
+- **Sass imports/partials**: These are files prefixed with an underscore (`_`) or explicitly marked as `isImport: true` in the package's `package.js` file, like this:
 
-The imports are intended to keep shared mixins and variables for your project,
-or to allow your package to provide several components which your package's
-users can opt into one by one.
+```javascript
+api.addFiles('x.scss', 'client', { isImport: true });
+```
 
-Each compiled source file produces a separate CSS file.  (The
-`standard-minifiers` package merges them into one file afterwards.)
+Each compiled source file generates a separate CSS file, which is then merged into one by the `standard-minifiers` package.
 
 ### Importing
```

Comment:
Looking at the changes here, this requires a migration guide to be included here.

Reply:
@illusionfield regarding migration guide:
- Add short description of difference between `@use` and `@forward` with reference to the docs
- Run mdlinter over the file (Bold fonts in headers are rudimentary)
- The rest looks good 👍 let's attach it to this PR